.gitignore

```diff
@@ -0,0 +1,3 @@
+*~
+*.pyc
+*.o
```

Comment:
I would not expect .o files to ever appear here. Did you observe them somewhere?

Reply:
No